libduel.cpp

```diff
@@ -4792,6 +4804,9 @@ int32 scriptlib::duel_majestic_copy(lua_State *L) {
 }
 
 static const struct luaL_Reg duellib[] = {
+	/////zdiy/////
+	{ "GetRandomNumber", scriptlib::duel_get_random_number },
```

Comment:
Put this line at the bottom of extra function block, right below `SetSummonCancelable`